### ISSUE TITLE
unable to log into ansible

### ISSUE THREAD

**Issue by dsbc4534** [spurin/diveintoansible-lab#88]
Hi, Mr. Spurin
I run out of ideas to troubleshoot on this issue.  I followed your instruction on installing Ansible lab on O'Reilly course.  I did get the point where I need to use ansible/password to log into the lab environment.  it  shows "login incorrect"
Here is my env setting
# Shared config volume
CONFIG=/host_mnt/C/Users/O425870/diveintoansible-lab/config

# Shared home directories
ANSIBLE_HOME=/host_mnt/C/Users/O425870/diveintoansible-lab/ansible_home

Note that I am using a company laptop.  Normally log into windows with O425870 as user.  But this user access is limited.  I do have admin right with O425870a account.  So I run the docker installation with admin right and launch it as admin as well.  otherwise it won't install or run.  then I launched command with admin account.  I was able to open Web Terminal Connectivity for Ansible Control Host, but not able to log into it with ansible/password.   I did try to run browser with admin account as well, same issue.  I did try Microsoft edge and google chrome.  Both failed.   oh, I also did grant O425870a full access to folder O425870 under user.

Regards,

**Comment by spurin** [comment]
Hiya,

Okay, so firstly, would you please try reverting the .env file to the default.  

With the latest version of the lab, you now shouldn’t need to edit this.

Hopefully this is all that this issue was as the format, is specific on windows.

If not, do let me know and we’ll look at another way of running the lab via virtualbox (I have a backup pre-created image for situations like this where work setups etc, may be causing problems with the docker security model).

Best Regards


James Spurin

**Comment by dsbc4534** [comment]
Thank you for your quick reply.
I did use the .env from the top of the page and downloaded updated yaml file.
# Shared config volume
CONFIG=./config

# Shared home directories
ANSIBLE_HOME=./ansible_home

Error message shows 
C:\Users\O425870\diveintoansible-lab>docker-compose up
yaml: line 170: mapping values are not allowed in this context

Using old yaml file I get
* error decoding 'Volumes[0]': invalid spec: :/config: empty section between colons

Regards,

**Comment by spurin** [comment]
Could you send me the .env and the docker-compose.yaml you’re using as this is a very strange error, there isn’t 170 lines in the docker-compose.yaml so I’m bit lost on what the problem is here.

My email is my ***@***.*** (substitute accordingly).

I’ll then check it and let you know where it’s going wrong.

Thanks


James

**Comment by dsbc4534** [comment]
Hi, Mr. Spurin
I got it up and running.  I am so sorry that I did not pay attention to file extensions when I tried to download the default .env file and yaml file from your git page.  So that 170 line must have been pointing to an HTML file.  Since then, it is working.  I am able to log in and completed the SSH lesson on O'Reilly.  
Thank you for your time and attention on this matter.  
By the way,  the google cloud shell works perfectly as well.

Regards,
